js/package.json

```diff
@@ -102,7 +118,8 @@
   "homepage": "https://github.com/langchain-ai/langsmith-sdk#readme",
   "dependencies": {
     "@types/uuid": "^10.0.0",
-    "commander": "^10.0.1",
+    "chalk": "^4.1.2",
```

Comment:
These new deps are just for test reporting, if desired we could split reporters into a separate package